pkg/cmd/openshift-kube-apiserver/server.go

```diff
@@ -43,6 +43,7 @@ func RunOpenShiftKubeAPIServerServer(kubeAPIServerConfig *kubecontrolplanev1.Kub
 	kubeDefaultOffAdmission := options.DefaultOffAdmissionPlugins
 	options.DefaultOffAdmissionPlugins = func() sets.String {
 		kubeOff := kubeDefaultOffAdmission()
+		kubeOff.Insert(originadmission.DefaultOffPlugins.List()...)
```

Comment:
@deads2k I put the default off here, since changing the other wasn't what we were expecting.

Reply:
> @deads2k I put the default off here, since changing the other wasn't what we were expecting.

yeah, this is better.